include/InputManager.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Camera.h"
+#include <GLFW/glfw3.h>
+
+class InputManager
+{
+public:
+	GLFWwindow* window;
+	Camera* camera;
+	
+	InputManager(GLFWwindow *window, Camera* cam);
+	const void CheckInput();
+	
+private:
+	float cursorX = 0;
+	float cursorY = 0;
+	
+	const void KeyInput();
+	const void MouseInput();
```

Comment:
You probably want:
`void MouseInput() const;`
as opposed to
`const void MouseInput();`

The first makes MouseInput() a const function.
The second means that MouseInput() returns a const void.

Reply:
I'm setting some values. guess it can't be const
cursorX = xpos;
cursorY = ypos;